package.json

```diff
@@ -1,53 +1,46 @@
 {
   "name": "opendistro-alerting",
-  "version": "1.11.0.2",
+  "version": "1.12.0.0",
   "description": "Kibana Alerting Plugin",
   "main": "index.js",
   "license": "Apache-2.0",
   "homepage": "https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin",
   "kibana": {
-    "version": "7.9.1",
-    "templateVersion": "6.3.3"
+    "version": "7.10.0"
   },
```

Comment:
I believe you can remove all Kibana-related versioning logic here, since it is maintained in the `kibana.json` manifest file. See [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/329)

Reply:
Thanks for reviewing! Yes, you are right, I have tried to remove it once, and it works. 
I also notice that when running the build script, the `"kibana": {"version": "7.10.0"}` will be added automatically to `package.json` in the artifact. This is the only reason I preserved it. Anyway, I can remove it.

Reply:
I removed the Kibana version in `package.json`, at least people don't need to change too many versions during version upgrading.